dist/assets/css/main.css

```diff
@@ -1096,6 +1096,19 @@ i.icon.burger span {
   max-width: none;
 }
 
+.jargon-p {
```

Comment:
Perhaps more descriptive and common name would be "tooltip". Any developer reading that would know what this bit of CSS is for. "Jargon" could mean anything.